### PR TITLE
Docs: Refactor and simplify our docs

### PR DESCRIPTION
**Pull request by ericholscher**
I’ve mostly focused on the first two sections of the docs, and tried to
make them a lot simpler and less technical. The goal here is for more users
to be able to understand product features instead of implementation at the
developer level.

I’ve also added a bit more sales and branding content that I think is
important for folks to understand what exactly RTD is. I think this helps
us communicate and sell the message of what we’re building, and hopefully
get aligned users on our side.

Refs #7055 -- I pulled those changes in too

## Review

The best way to review this is going to be looking at the rendered docs, I think: https://docs--7052.org.readthedocs.build/en/7052/

It's a lot of content changes, and it's pretty difficult to review in the diff viewer.

The most interesting pages:

* Features: https://docs--7052.org.readthedocs.build/en/7052/features.html
* About: https://docs--7052.org.readthedocs.build/en/7052/about.html
* Downloadable formats: https://docs--7052.org.readthedocs.build/en/7052/downloadable-formats.html
* Build process: https://docs--7052.org.readthedocs.build/en/7052/builds.html
* Hosting: https://docs--7052.org.readthedocs.build/en/7052/hosting.html

### CODE REVIEW

File: docs/features.rst [hunk 1]
```diff
@@ -1,70 +1,91 @@
...
+Learn more about :doc:`/custom_domains`.
 
-Canonical URLs give your docs better search performance, by pointing all URLs to one version. This also helps to solve the issues around users landing on outdated versions of documentation.
+Versioned Documentation
+-----------------------
 
-More information can be found in the :doc:`guides/canonical` page.
+We support multiple versions of your documentation,
+so that users can find the exact docs for the version they are using.
+We build this on top of the Version Control system that you're already using.
```
Comment (agjohnson) [inline docs/features.rst:36]:
```suggestion
We build this on top of the version control system that you're already using.
```

File: docs/features.rst [hunk 2]
```diff
@@ -1,70 +1,91 @@
...
+Each Version on Read the Docs is just a tag or branch in your repository.
```
Comment (agjohnson) [inline docs/features.rst:37]:
```suggestion
Each version on Read the Docs is just a tag or branch in your repository.
```

File: docs/features.rst [hunk 3]
```diff
@@ -1,70 +1,91 @@
...
-+============+============+===========+============+===========+
-| Tags       |    Yes     |    Yes    |   Yes      |    No     |
-+------------+------------+-----------+------------+-----------+
-| Branches   |    Yes     |    Yes    |   Yes      |    No     |
-+------------+------------+-----------+------------+-----------+
-| Default    |    master  |   default |            |    trunk  |
-+------------+------------+-----------+------------+-----------+
+We provide search across all the projects that we host.
+This actually comes in two different search experiences:
+Dashboard Search on the Read the Docs dashboard and in-doc search on the documentation sites, using your own theming and our search results.
```
Comment (agjohnson) [inline docs/features.rst:67]:
```suggestion
dashboard search on the Read the Docs dashboard and in-doc search on documentation sites, using your own theme and our search results.
```

File: docs/features.rst [hunk 4]
```diff
@@ -1,70 +1,91 @@
...
 
+We offer a number of search features:
 
-PDF Generation
---------------
+* Search across :doc:`subprojects </subprojects>`
+* Search results land on the exact content you were looking for
+* Search across projects you have access to (|com_brand|)
```
Comment (agjohnson) [inline docs/features.rst:73]: Perhaps "available on |com_brand|" for clarification?

File: docs/guides/searching-with-readthedocs.rst [hunk 5]
```diff
@@ -1,48 +1,20 @@
 Searching with Read the Docs
 ============================
 
-Read the Docs uses Elasticsearch to provide a better search experience.
+Read the Docs uses :doc:`/server-side-search` to provide a better search experience.
```
Comment (agjohnson) [inline docs/guides/searching-with-readthedocs.rst:4]: A holdover from the original text, but "better" doesn't refer to anything here.

File: docs/index.rst [hunk 6]
```diff
@@ -4,29 +4,34 @@ Read the Docs: Documentation Simplified
...
     so having a 1.0 version of your docs and a 2.0 version
     of your docs is as easy as having a separate branch or tag in your version control system.
+    Read more about :doc:`/versions`.
 
-Free and open source
-    Read the Docs is free and open source and hosts documentation
-    for nearly 100,000 large and small open source projects
+Open Source and User Focused |:heartbeat:|
+    Our code is free and `open source <https://github.com/readthedocs/>`_.
+    :doc:`Our company </about>` is boostrapped and 100% user focused.
```
Comment (agjohnson) [inline docs/index.rst:26]:
```suggestion
    :doc:`Our company </about>` is bootstrapped and 100% user focused.
```
I'm not sure of a better way to say bootstrapped here, but it feels a bit unnecessary in this context, and "100% user focused" is maybe too fluffy of a statement.

What thi is

File: docs/index.rst [hunk 7]
```diff
@@ -4,29 +4,34 @@ Read the Docs: Documentation Simplified
...
+    readthedocs.org hosts documentation for over 100,000 large
```
Comment (agjohnson) [inline docs/index.rst:27]:
```suggestion
    |org_brand| hosts documentation for over 100,000 large 
```

File: docs/index.rst [hunk 8]
```diff
@@ -4,29 +4,34 @@ Read the Docs: Documentation Simplified
...
-Free and open source
-    Read the Docs is free and open source and hosts documentation
-    for nearly 100,000 large and small open source projects
+Open Source and User Focused |:heartbeat:|
+    Our code is free and `open source <https://github.com/readthedocs/>`_.
+    :doc:`Our company </about>` is boostrapped and 100% user focused.
+    readthedocs.org hosts documentation for over 100,000 large 
+    and small open source projects,
     in almost every human and computer language.
+    readthedocs.com supports hundreds of organizations with product and internal  documentation.
```
Comment (agjohnson) [inline docs/index.rst:30]:
```suggestion
    |com_brand| supports hundreds of organizations with product and internal documentation.
```

File: docs/about.rst [hunk 9]
```diff
@@ -0,0 +1,27 @@
+About Read the Docs
+===================
+
+Read the Docs is a C Corporation registered in Oregon.
+Our bootstrapped company is owned and fully controlled by the founders,
+and fully funded by our customers and advertisers.
+This allows us to **focus 100% on our users**. 
+
+We primarily make money in two ways:
```
Comment (agjohnson) [inline docs/about.rst:9]:
```suggestion
We have two main sources of revenue:
```

File: docs/about.rst [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+* |com_brand| - where we `provide <https://readthedocs.com>`__ a valuable paid service to companies.
+* |org_brand| - where we provide a free service to the open source community, funded via :doc:`/advertising/ethical-advertising`.
+
+We believe that having both paying customers and ethical advertising is the best way to create a sustainable platform for our users.
+We have built something that we expect to last a long time,
+and we are able to make decisions based on the best interest of our community and customers only.
```
Comment (agjohnson) [inline docs/about.rst:16]:
```suggestion
and we are able to make decisions based only on the best interest of our community and customers.
```

File: docs/about.rst [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+
+All of the source code for Read the Docs is open source.
+You are welcome to :doc:`contribute </contribute>` the features you want or run your own instance.
+We should note that we generally only support our hosted versions as a matter of :doc:`philosophy </open-source-philosophy>`.
```
Comment (agjohnson) [inline docs/about.rst:20]:
```suggestion
We should note that we generally only support our hosted versions as a matter of :doc:`our philosophy </open-source-philosophy>`.
```

Or "open source philosophy" maybe? This currently sounds like philosophy in general.

File: docs/automatic-redirects.rst [hunk 12]
```diff
@@ -4,29 +4,44 @@ Automatic Redirects
...
+---------------------
+
+You can link to a specific page and have it redirect to your default version.
+This is done with the ``/page/`` URL.
+You can reach this page by going to https://docs.readthedocs.io/page/automatic-redirects.html.
+
+This allows you to create links that are always up to date.
+
+Another way to handle this is the *latest* version.
+You can set your ``latest`` version to a specific version and just always link to latest.
```
Comment (agjohnson) [inline docs/automatic-redirects.rst:17]: We use 3 way of formatting for "latest" here, and could consolidate for clarity

```suggestion
Another way to handle this is the ``latest`` version.
You can set your ``latest`` version to a specific version and just always link to ``latest``.
```

Or all bold, either way works

File: docs/builds.rst [hunk 13]
```diff
@@ -70,35 +37,32 @@ It should also allow you to take advantage of certain things that happen during
 The first step of the process is that we check out your code from the repository you have given us.
 If the code is already checked out, we update the copy to the branch that you have specified in your project's configuration.
 
-Then we build the proper backend code for the type of documentation you've selected.
+Then we build the proper backend code for the type of documentation you've selected,
+this is done inside a :ref:`Docker image <builds:Docker images>`.
```
Comment (agjohnson) [inline docs/builds.rst:41]:
```suggestion
this is done inside a :ref:`Docker container <builds:Docker images>`.
```

File: docs/builds.rst [hunk 14]
```diff
@@ -108,79 +72,39 @@ An example in code:
...
-.. csv-table::
-   :header-rows: 1
+.. csv-table:: Environment Variables
+   :header: Environment variable, Description, Example value
+   :widths: 15, 10, 30
+    
+   ``READTHEDOCS``, Whether the build is running inside RTD, ``True``
+   ``READTHEDOCS_VERSION``, The RTD name of the version which is being built, ``latest``
+   ``READTHEDOCS_PROJECT``, The RTD slug of the project which is being built, ``my-example-project``
+   ``READTHEDOCS_LANGUAGE``, The RTD language slug of the project which is being built, ``en``
```
Comment (agjohnson) [inline docs/builds.rst:87]: Also a good use for a standard definition list, but both work.

File: docs/guides/searching-with-readthedocs.rst (hunk 5 above)
Comment (ericholscher) [reply docs/guides/searching-with-readthedocs.rst:4]: I think it's comparing it to the default search builtin to the tooling.

**Comment by stsewd** [comment]
Would be good to document that we serve a default 404, sitemap and robots pages (and some of them can be overridden). I think they are hidden in the guides now.

File: docs/downloadable-formats.rst [hunk 15]
```diff
@@ -0,0 +1,40 @@
+Downloadable Formats
```
Comment (davidfischer) [inline docs/downloadable-formats.rst:1]: Would "Downloadable Documentation" be a better name?

File: docs/features.rst [hunk 16]
```diff
@@ -1,70 +1,91 @@
...
---------------------
+Custom Domains & White Labeling
+-------------------------------
 
-Read the Docs itself is localized, and we support documentation translated into multiple languages.
-Read more on the :doc:`localization` and :doc:`development/i18n` pages.
+When you import a project to Read the Docs,
+we assign you a URL based on your project name.
+You are welcome to use this URL,
+but we also fully support custom domains (``CNAME's``) for all our documentation projects.
```
Comment (davidfischer) [inline docs/features.rst:27]: Maybe not in this PR but getting rid of the term CNAME (unless referring specifically to DNS) is probably good. We should call them "custom domains". This is especially true now that we explicitly support root domains which are not CNAMEs.

File: docs/intro/getting-started-with-mkdocs.rst [hunk 17]
```diff
@@ -62,6 +62,11 @@ you can start using Read the Docs by :doc:`importing your docs </intro/import-gu
 
 .. _install MkDocs: https://www.mkdocs.org/#installation
 
+.. warning::
```
Comment (davidfischer) [inline docs/intro/getting-started-with-mkdocs.rst:65]: This banner is good but it's a little close to the image which causes some awkwardness.

Screen Shot 2020-05-08 at 2 43 15 PM

File: docs/index.rst [hunk 18]
```diff
@@ -4,29 +4,34 @@ Read the Docs: Documentation Simplified
...
    :description lang=en: Automate building, versioning, and hosting of your technical documentation continuously on Read the Docs.
 
-
 `Read the Docs`_ simplifies software documentation
-by automating building, versioning, and hosting of your docs for you.
+by building, versioning, and hosting of your docs, automatically.
 Think of it as *Continuous Documentation*.
 
-Never out of sync
+Never out of sync |:arrows_counterclockwise:|
```
Comment (davidfischer) [inline docs/index.rst:11]: :heart:

File: docs/conf.py [hunk 19]
```diff
@@ -101,6 +102,10 @@ def get_version():
     'confval': 'tooltip',  # for custom object
 }
 
+rst_epilog = """
```
Comment (davidfischer) [inline docs/conf.py:105]: This is the section causing issues with docs-lint. I'm not sure what a great solution is yet.

File: docs/features.rst (hunk 16 above)
Comment (ericholscher) [reply docs/features.rst:27]: Sure, I just want it to be explicit and searchable. It's called custom domains in the primary content, but I think its useful to mention.

File: docs/downloadable-formats.rst (hunk 15 above)
Comment (ericholscher) [reply docs/downloadable-formats.rst:1]: Perhaps, it has nice alliteration

File: docs/guides/searching-with-readthedocs.rst (hunk 5 above)
Comment (agjohnson) [reply docs/guides/searching-with-readthedocs.rst]: Yeah, it's just not clear to the reader that we're referring to the built in sphinx search

File: docs/features.rst (hunk 16 above)
Comment (agjohnson) [reply docs/features.rst:27]: I'm +1 on removing CNAME here as well, this usage is confusing as it's a DNS term.

File: docs/conf.py (hunk 19 above)
Comment (ericholscher) [reply docs/conf.py:105]: I got it fixed up by ignoring them for now.

**Comment by ericholscher** [comment]
> Would be good to document that we serve a default 404, sitemap and robots pages (and some of them can be overridden). I think they are hidden in the guides now.

I added the Hosting Features page to address this 👍

File: docs/faq.rst [hunk 20]
```diff
@@ -243,44 +260,45 @@ and start suggesting the latest (or newer) one.
...
 
-MkDocs needs the ``robots.txt`` to be at the directory defined at `docs_dir`_ config.
+We deploy readthedocs.org from the `rel` branch in our GitHub repository.
+You can see the latest commits that have been deployed by looking on GitHub: https://github.com/readthedocs/readthedocs.org/commits/rel
 
-.. _Google's docs: https://support.google.com/webmasters/answer/6062608
-.. _html_extra_path: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_extra_path
-.. _docs_dir: https://www.mkdocs.org/user-guide/configuration/#docs_dir
+We also keep an up-to-date :doc:`changelog </changelog>`.
+>>>>>>> origin/davidfischer/minor-faq-rework
```
Comment (stsewd) [inline docs/faq.rst:304]: This shouldn't be here :eyes:

File: tox.ini [hunk 21]
```diff
@@ -51,7 +51,7 @@ description = run linter (rstcheck) to ensure there aren't errors on our docs
 deps = -r{toxinidir}/requirements/docs-lint.txt
 changedir = {toxinidir}/docs
 commands =
-    rstcheck -r .
+    rstcheck -r --ignore-substitutions org_brand,com_brand .
```
Comment (stsewd) [inline tox.ini:54]: These should be in https://github.com/readthedocs/readthedocs.org/blob/master/docs/.rstcheck.cfg

File: docs/downloadable-formats.rst [hunk 22]
```diff
@@ -0,0 +1,40 @@
...
+    * `Zipped HTML`_ 
+    
+.. _PDF: https://docs.readthedocs.io/_/downloads/en/latest/pdf/
+.. _ePub: https://docs.readthedocs.io/_/downloads/en/latest/epub/
+.. _Zipped HTML: https://docs.readthedocs.io/_/downloads/en/latest/htmlzip/
+
+Use cases
+---------
+
+This functionality is great for anyone who needs documentation when they aren't connected to the internet
```
Comment (humitos) [inline docs/downloadable-formats.rst:36]:
```suggestion
This functionality is great for anyone who needs documentation when they aren't connected to the internet.
```

File: docs/faq.rst [hunk 23]
```diff
@@ -93,6 +69,30 @@ If such libraries are installed via ``setup.py``, you also will need to remove a
 
 .. _autodoc_mock_imports: http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports
 
+
+How do I change behavior when building with Read the Docs?
+----------------------------------------------------------
+
+When RTD builds your project, it sets the :envvar:`READTHEDOCS` environment
+variable to the string `True`. So within your Sphinx :file:`conf.py` file, you
```
Comment (humitos) [inline docs/faq.rst:77]:
```suggestion
variable to the string ``'True'``. So within your Sphinx :file:`conf.py` file, you
```

File: docs/faq.rst [hunk 24]
```diff
@@ -137,40 +139,60 @@ http://docs.celeryproject.org/projects/kombu/en/latest/
...
+You can specify an exact path to your documentation using a Read the Docs :doc:`config-file/index`.
+
 
 I want to use the Blue/Default Sphinx theme
 -------------------------------------------
 
-We think that our theme is badass, and better than the default for many reasons. Some people don't like change though :), so there is a hack that will let you keep using the default theme. If you set the ``html_style`` variable in your ``conf.py``, it should default to using the default theme. The value of this doesn't matter, and can be set to ``/default.css`` for default behavior.
+We think that our theme is badass,
+and better than the default for many reasons.
+Some people don't like change though :),
```
Comment (humitos) [inline docs/faq.rst:160]:
```suggestion
Some people don't like change though :smile:,
```

Now that you have installed the emojis :wink:

File: docs/faq.rst [hunk 25]
```diff
@@ -209,30 +232,24 @@ If you want to place your package in a different directory or have multiple
 python packages in the same project, then create a pip requirements file. You
```
Comment (humitos) [inline docs/faq.rst:232]: In the previous paragraph, we should probably change

```
in your documentation is to use the *Install your project
inside a virtualenv using setup.py install* option in the admin panel of
your project
```

to link to the config file option.

File: docs/faq.rst [hunk 26]
```diff
@@ -243,44 +260,45 @@ and start suggesting the latest (or newer) one.
...
-Sphinx uses `html_extra_path`_ option to add static files to the output.
-You need to create a ``robots.txt`` file and put it under the path defined in ``html_extra_path``.
 
-MkDocs
-++++++
+What commit of Read the Docs is in production?
+----------------------------------------------
 
-MkDocs needs the ``robots.txt`` to be at the directory defined at `docs_dir`_ config.
+We deploy readthedocs.org from the `rel` branch in our GitHub repository.
```
Comment (humitos) [inline docs/faq.rst:300]:
```suggestion
We deploy readthedocs.org from the ``rel`` branch in our GitHub repository.
```

File: docs/features.rst [hunk 27]
```diff
@@ -1,70 +1,91 @@
...
-that we've inferred from the tags and branches in your source control system (according to
-the support matrix below). On the Versions page you can tell us which versions you'd like us
-to build docs for, whether each should be public, protected, or private, and what the default
-version should be (we'll redirect there when someone hits your main project page, e.g.,
-http://my-project.rtfd.org/).
+Learn more about :doc:`/versions`.
 
-Version Control Support Matrix
--------------------------------
+Downloadable Formats
```
Comment (humitos) [inline docs/features.rst:46]: Remember to use the same naming to whatever you decide in a previous comment.

File: docs/index.rst (hunk 18 above)
Comment (humitos) [reply docs/index.rst:11]: Emojis are really good here.

BTW, I don't like how this renders in the new version of the theme, but I suppose that's a bug.

File: docs/server-side-search.rst [hunk 28]
```diff
@@ -0,0 +1,61 @@
...
+Read the Docs provides full-text search across all of the pages of all projects,
+this is powered by Elasticsearch_.
+You can search all projects at https://readthedocs.org/search/,
+or search only on your project from the :guilabel:`Search` tab of your project.
+
+We override the default search engine of your project with our search engine
+to provide you better results within your project.
+We fallback to the built-in search engine from your project
+if our search engine doesn't return any results,
+just in case we missed something :)
```
Comment (humitos) [inline docs/server-side-search.rst:13]:
```suggestion
just in case we missed something :smile:
```

**Pull request merged**